src/control/cmd/dmg/main.go

```diff
@@ -121,6 +122,19 @@ func exitWithError(log logging.Logger, err error) {
 	os.Exit(1)
 }
 
+func writeManPage(wr io.Writer) {
+	var opts cliOptions
+	p := flags.NewParser(&opts, flags.Default)
+	p.ShortDescription = "Administrative tool for managing DAOS clusters"
+	p.Usage = "dmg [OPTIONS] [COMMAND][SUBCOMMAND]"
+	p.LongDescription = `dmg (DAOS Management) is a tool for connecting to DAOS servers
+for the purpose of issuing administrative commands to the cluster. dmg is
+provided as a means for allowing administrators to securely discovery and
```

Comment:
Typo: "discovery" -> "discover"

Reply:
Correct. I reworded that sentence a couple of times and this was left over.